src/go/mongolib/stats/stats.go

```diff
@@ -110,9 +111,15 @@ func (s *Stats) Queries() Queries {
 	s.RLock()
 	defer s.RUnlock()
 
+	keys := GroupKeys{}
+	for key := range s.queryInfoAndCounters {
+		keys = append(keys, key)
+	}
+	sort.Sort(keys)
+
 	queries := []QueryInfoAndCounters{}
-	for _, v := range s.queryInfoAndCounters {
-		queries = append(queries, *v)
+	for _, key := range keys {
+		queries = append(queries, *s.queryInfoAndCounters[key])
 	}
 	return queries
```

Comment:
@cfsalguero queries were converted from unsorted map to sorted slice, so the order of elements was unpredictable, by sorting them by key I've kept the order. Let me know if you want this different.